syntax = "proto3";

service EmailInfo{
 rpc GetEmailInfo(Person) returns (Emailaddress) {}
}
 
message Person {
  string FirstName = 1;
  string LastName = 2;
}

message Emailaddress {
  string Email = 1;
}

/* 
python -m grpc_tools.protoc --proto_path=. ./addressbook.proto --python_out=. --grpc_python_out=.
*/
